src/workerd/server/BUILD.bazel

```diff
@@ -11,17 +11,21 @@ config_setting(
 wd_cc_binary(
     name = "workerd",
     srcs = ["workerd.c++"],
+    env = {
+        "LLVM_SYMBOLIZER": "llvm-symbolizer",
```

Comment:
I guess this would only have an effect if `workerd` is executed using `bazel run`? To be honest I've never done that, it doesn't seem like what most people would do.

Can we instead change `symbolizer.c++` to fall back to this default if the env var isn't set?